### PR TITLE
Adding bat support

### PR DESCRIPTION
**Pull request by casonadams**
Default to `bat` if installed otherwise fall back to original logic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,6 +234,11 @@ let g:nv_ignore_pattern = ['summarize-*', 'misc*']
 let g:nv_expect_keys = []
 ```
 
+### bat config
+
+If bat is in the PATH then it will be used. To configure the theme
```
Comment (alok) [inline README.md:239]: `bat`

File: README.md [hunk 2]
```diff
@@ -234,6 +234,11 @@ let g:nv_ignore_pattern = ['summarize-*', 'misc*']
...
+use the bat ENV var `export BAT_THEME="gruvbox"`
```
Comment (alok) [inline README.md:240]: just say it will use `$BAT_THEME` if it's set

File: print_lines.py [hunk 3]
```diff
@@ -4,22 +4,47 @@
...
+        lines = opcount(file)
+        cmd = [
+            "bat",
+            "--style=numbers",
+            "--color=always",
+            "--highlight-line={}".format(line),
+        ]
+        if (line > (height / 2)) and (lines >= height):
+            insert = "--line-range={}:{}".format(int(line - (height / 2)), lines)
+            cmd.append(insert)
```
Comment (alok) [inline print_lines.py:35]: make sure this code is run through `black` by the way

**Review by alok — CHANGES_REQUESTED** [review]
see comments

File: print_lines.py (hunk 3 above)
Comment (casonadams) [reply print_lines.py:35]: This is the result of a `black` format.  What should I expect the output to be?

**Review by alok — APPROVED** [review]

**Pull request merged**